src/modules/import/file.js

```diff
@@ -10,7 +10,7 @@ goog.provide('ngeo.File');
  * @ngInject
  */
 ngeo.File = function($q, $http, $window, gettext) {
-  var fileReader, canceler;
+  let fileReader, canceler;
```

Comment:
Great !
Very nice initiative and a nice step forward :)
Did you test to use the `Class` uppon a Controller for example ?
Maybe we could automate with lebab the refactoring from es5 to es6 classes, taking care of closure and doc annotations.

Maybe in a next step.